api/mfx_dispatch/linux/mfxparser.cpp

```diff
@@ -131,43 +137,54 @@ void parse(const char* file_name, std::list<PluginInfo>& plugins)
 #else
   const char* mode = "r";
 #endif
-  char line[PATH_MAX];
-  PluginInfo plg;
 
   FILE* file = fopen(file_name, mode);
   if (!file)
     return;
 
-  while(fgets(line, PATH_MAX, file)) {
-    char* p = skip(line);
+  char c_line[PATH_MAX];
+  PluginInfo plg;
+  std::string line;
+
+  while(fgets(c_line, PATH_MAX, file))
+  {
+    line = c_line;
+
+    skip(line);
+    strip(line);
 
-    if (strchr(";#", *p)) {
+    if (line.find_first_of(";#") != std::string::npos)
```

Comment:
It will be found even if not at the first place, unlike it was before

Reply:
right, it should be ```find_first_not_of```

Reply:
Hm. Then  find_first_not_of != 0.
But no reason to search for not a comment symbol if you only need to check first in the line.